tsl/test/expected/size_utils.out

```diff
@@ -0,0 +1,71 @@
+-- This file and its contents are licensed under the Apache License 2.0.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-APACHE for a copy of the license.
+CREATE TABLE compressed(time timestamptz NOT NULL);
```

Comment:
could you name the table as compressed_tbl or something similar to indicate it is a table.